iocage/iocage.8

```diff
@@ -0,0 +1,1646 @@
+.It Op Fl -http | h
+Change
+.Op Fl s | -server
+to define an HTTP server instead of the default FTP.
+.It Op Fl -noeol | NE
+Disable EOL checking upstream.
+.It Op Fl -noupdate | NU
+Disable updating the fetch item to the latest patch level.
+.It Op Fl -noverify | NV
+Disable verifying the SSL cert for HTTP fetching.
+.It Op Fl -password | p
+Add a password, if required.
+.It Op Fl -plugin-file | P
+Specify which plugin file to use.
+.It Op Fl -plugins
+List all available plugins for creation.
+.It Op Fl -release | r
+Define the
+.Fx
+release to fetch.
+.It Op Fl -root-dir | d
+Specify the root directory containing all RELEASE files.
+.It Op Fl -server | s
+Define which FTP server to log into.
+.It Op Fl -update | U
+Update the fetch to the latest patch level.
+.It Op Fl -user | u
+Define the user.
+.It Op Fl -verify | V
+Enable verifying the SSL cert for HTTP fetching.
+.El
+.Pp
+Examples:
+.Bl -item
+.It
+.Nm
+fetch release=10.1-RELEASE
+.El
+.Pp
+.Cm fetch
+is also used to update already downloaded releases.
+To update a local release already present in
+.Nm
+.Po
+.Cm iocage list -r
+.Pc ,
+run:
+.Bl -item
+.It
+.Nm
+fetch release=10.1-RELEASE
+.El
+.Pp
+This example applies the latest patches to 10.1-RELEASE base.
+Newly created jails or basejails will automatically have the latest
+updates applied.
+.\" == FSTAB ==
+.It Cm fstab
+Manipulates the fstab settings of a specific jail.
+.Pp
+Options:
+.Bl -tag -width "[-r | --remove]"
+.It Op Fl -add | a
+Adds an entry to the specific jail
+.Pa fstab
+and mount it.
+.It Op Fl -edit | e
+Open the
+.Pa fstab
+file in the default editor.
+.It Op Fl -remove | r
+Remove an entry from a specific jail fstab and unmount it.
+.El
+.\" == GET ==
+.It Cm get
+Display the specified property.
+.Pp
+Options:
+.Bl -tag -width "[-H | -h | --header]"
+.It Op Fl -all | a
+Get all properties for the specified jail.
+.It Op Fl -header | H | h
+Used in scripting.
+Use tabs for separators.
+.It Op Fl -plugin | P
+Get the specified key for a plugin jail.
+If accessing a nested key, use "." as a separator.
+.It Op Fl -pool | p
+Get the currently activated zpool.
+.It Op Fl -recursive | r
+Get the specified property for all jails.
+.El
+.\" == IMPORT ==
+.It Cm import
+Import a specific jail image.
+Short UUIDs can be used, but do not specify the full filename, only the
+UUID.
+.\" == LIST ==
+.It Cm list
+List a specified dataset type.
+By default, all jails are listed.
+.Pp
+Options:
+.Bl -tag -width "[-H | -h | --header]"
+.It Op Fl -base | -release | b | r
+List all bases.
+.It Op Fl -header | H | h
+Used in scripting.
+Use tabs for separators.
+.It Op Fl -http | h
+Tells
+.Op Fl R | -remote
+to use HTTP.
+.It Op Fl -long | l
+Shows the full UUID and ip4 address.
+.It Op Fl -plugins | P
+Shows available plugins.
+.It Op Fl -remote | R
+Shows available RELEASE options for remote.
+.It Op Fl -template | t
+Lists all templates.
+.El
+.\" == MIGRATE ==
+.It Cm migrate
+Migrate all jails to a new jail format.
```

Comment:
This will only migrate `iocage_legacy`'s `develop` branch users, this is not meant for stable -> new iocage. Thta needs to be explicitly mentioned.